controllers/articles.controllers.js

```diff
@@ -19,3 +20,13 @@ exports.getAllArticles = (request, response, next) => {
     })
     .catch(next);
 };
+exports.patchArticleVotes = (request, response, next) => {
+  const { article_id } = request.params;
+  const articleBody = request.body;
+
+  updateArticleVotes(article_id, articleBody)
+    .then((updatedArticle) => {
+      response.status(200).send({ updatedArticle });
```

Comment:
this response should be send back on the key of 'article' - this key could cause you issues in the future!